docs/pyretriever.rst

```diff
@@ -189,24 +204,52 @@ A description of default parameters mentioned above:
 
 .. code-block:: python
 
-  compile   (Bool): Setting True recompiles scripts upon installation.
+  compile         (Bool): Setting True recompiles scripts upon installation.
+
+  database_name (String): Specify database name. For postgres, mysql users.
+
+  debug           (Bool): Setting True helps in debugging in case of errors.
 
-  database_name(String): Specify database name. For postgres, mysql users.
+  file          (String): Enter file_name for database. For msaccess, sqlite users.
 
-  debug     (Bool): Setting True helps in debugging in case of errors.
+  host          (String): Specify host name for database. For postgres, mysql users.
 
-  file      (String): Enter file_name for database. For msaccess, sqlite users.
+  password      (String): Specify password for database. For postgres, mysql users.
 
-  host      (String): Specify host name for database. For postgres, mysql users.
+  port             (Int): Specify the port number for installtion. For postgres, mysql users.
 
-  password  (String): Specify password for database. For postgres, mysql users.
+  quiet           (Bool): Setting True minimizes the console output.
 
-  port      (Int): Specify the port number for installtion. For postgres, mysql users.
+  table_name    (String): Specify the table name to install.
+
+  use_cache       (Bool): Setting False reinstall scripts if it is already installed.
+
+  user          (String): Specify the user_name. For postgres, mysql users.
+  
+Examples to Installing Datasets:
+
+Here, we are installing the CSV file to the dataset ``wine-composition`` to our current-working directory.
+
+.. code-block:: python
+
+  rt.install_csv("wine-composition")
+
+  => Installing wine-composition
+
+  Downloading wine.data...
+  100%  0 seconds Progress: 178/178 rows inserted into ./wine_composition_WineComposition.csv totaling 178
+
+The installed file is called ``wine_composition_WineComposition.csv``
+
+Similarly, we can download the JSON file to any available dataset as follows:
+
+.. code-block:: python
 
-  quiet     (Bool): Setting True minimizes the console output.
+  rt.install_json("wine-composition")
 
-  table_name(String): Specify the table name to install.
+  => Installing wine-composition
 
-  use_cache (Bool): Setting False reinstall scripts if it is already installed.
+  Progress: 178/178 rows inserted into ./wine_composition_WineComposition.json totaling 17
 
-  user      (String): Specify the user_name. For postgres, mysql users.
+The JSON file to the dataset ``wine-composition`` called ``wine_composition_WineComposition.json``
```

Comment:
Change to:
The ``wine-composition`` dataset is now installed as a JSON file called ``wine_composition_WineComposition.json`` in our current-working directory.